README.md

```diff
@@ -0,0 +1,35 @@
+# Unimus Backup Extractor
+
+This repository contains the Unimus Backup Extractor.  
+This extractor is used to create local copies of unimus device backups. 
+
+### TL;DR
+Requirements: `bash`, `jq`, `base64`.
+```text
+./unimus-backup-exporter.sh
+``` 
+
+### How does the exporter work?
```

Comment:
in this section, describe how this works, uses Unimus API, exports backups either `latest` or `all`, etc.